packages/quantic/force-app/main/default/lwc/quanticQuickviewContent/quanticQuickviewContent.css

```diff
@@ -0,0 +1,17 @@
+@import 'c/quanticResultQuickview';
+@import 'c/quanticResultActionStyles';
+
+.iframe-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
These two lines can be replaced by the following SLDS CSS classes: `slds-grid` and `slds-grid_vertical`
https://www.lightningdesignsystem.com/utilities/grid/#Top-to-Bottom